packages/apps/web-viewer-test/tsconfig.json

```diff
@@ -1,11 +1,7 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
+    "target": "es2020",
```

Comment:
Nice! I was hoping to make this the default in itwin/build-tools/tsconfig-base but babel had issues parsing some of the new code. But should work with webpack 5, and I'd like to see if I can change the default for 3.2 if it doesn't cause a breaking change

Reply:
this one should be done sooner rather than later as well. The targets in this repo are scattered: https://github.com/iTwin/viewer/issues/133. Maybe we should wait until we decide if we can go 2020 across the board